### PR TITLE
release 0.0.1-rc.24-v1

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: migrate-base.js [hunk 1]
```diff
@@ -142,7 +142,11 @@ async function migrateBase ({ web3, spinner, confirm, opts, logTx, previousMigra
   if (Number(arcVersion.slice(-2)) >= 22) {
     await deploy(require('@daostack/arc/build/contracts/Redeemer.json'))
   }
-  await deploy(require('@daostack/arc/build/contracts/UGenericScheme.json'))
+  if (Number(arcVersion.slice(-2)) >= 24) {
```
Comment (orenyodfat) [inline migrate-base.js]: why this is needed ?
is that needed for each new arc version ?

File: migrate-dao.js [hunk 2]
```diff
@@ -372,7 +373,7 @@ async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logT
       tx = await genericSchemeSetParams.send({ nonce: ++nonce })
       await logTx(tx, 'Generic Scheme parameters set.')
       schemeNames.push('Generic Scheme')
-      schemes.push(UGenericScheme)
+      schemes.push(Number(arcVersion.slice(-2)) >= 24 ? UGenericScheme : GenericScheme)
```
Comment (orenyodfat) [inline migrate-dao.js]: why ?

File: prepare-release.sh [hunk 3]
```diff
@@ -3,42 +3,42 @@
 set -e
 source .env
 
+# # npm ci
+# echo "Installing NPM modules..."
```
Comment (orenyodfat) [inline prepare-release.sh]: why this is comment out ?

File: prepare-release.sh [hunk 4]
```diff
@@ -3,42 +3,42 @@
...
+# # migrating Arc version to ganache
+# echo "Migrating ganache..."
+# npm run migrate -- --force "$@"
+# done
+# # migrate kovan
+# echo "Migrating kovan..."
+# npm run migrate -- --gasPrice 10 --provider $kovan_provider --private-key $kovan_private_key "$@"
+# # migrate rinkeby
+# echo "Migrating rinkeby..."
+# npm run migrate -- --gasPrice 10 --provider $rinkeby_provider --private-key $rinkeby_private_key "$@"
```
Comment (orenyodfat) [inline prepare-release.sh]: why this is comment out ?

File: migrate-base.js (hunk 1 above)
Comment (ben-kaufman) [reply migrate-base.js]: Just when we have breaking changes

File: prepare-release.sh (hunk 3 above)
Comment (ben-kaufman) [reply prepare-release.sh]: fixed

**Pull request merged**